rpcutil/rpcutil.go

```diff
@@ -153,6 +158,16 @@ func (cr llCodecRequest) ReadRequest(args interface{}) error {
 		}
 	}
 
+	if cr.c.RunInputApplicators {
+		if err := applicator.Apply(args); err != nil {
+			llog.Error("error running applicator on args", cr.kv, llog.KV{"err": err})
```

Comment:
That's something that would pretty much immediately be discovered during tests though, so it's not likely we'll see it in production. And it is *possible* that we would want to have an applicator that returns an error not related to the types in the future, which this completely prevents us from sending to the client.